### ISSUE TITLE
`fetch_repo_data`  error

### ISSUE THREAD

**Issue by dhirschfeld** [prefix-dev/rattler-build#178]
I'm seeing an error from `fetch_repo_data`:
```
ERROR fetch_repo_data{cache_path=/home/user/.cache/rattler/cache/repodata}: rattler_repodata_gateway::fetch: error=repodata not found
 WARN fetch_repo_data{cache_path=/home/user/.cache/rattler/cache/repodata}: rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN fetch_repo_data{cache_path=/home/user/.cache/rattler/cache/repodata}: rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN fetch_repo_data{cache_path=/home/user/.cache/rattler/cache/repodata}: rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN fetch_repo_data{cache_path=/home/user/.cache/rattler/cache/repodata}: rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN fetch_repo_data{cache_path=/home/user/.cache/rattler/cache/repodata}: rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN fetch_repo_data{cache_path=/home/user/.cache/rattler/cache/repodata}: rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
```

Is this an actual error? The build seems to complete fine despite this.

**Comment by dhirschfeld** [comment]
I also see lots of similar warnings:
```
WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
 WARN rattler_networking::authentication_storage::storage: Error retrieving credentials for *.conda.anaconda.org: Platform secure storage failure: zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.secrets was not provided by any .service files, using fallback storage at ~/.rattler/rattler_auth_store.json
✔ Successfully updated the environment
```

**Comment by ruben-arts** [comment]
Oehf that is annoying. This is not an actual error thus should definitely be more silent. 

This is coming from rattler. [storage.rs](https://github.com/mamba-org/rattler/blob/6c908c5d59908bf613086caf31dd6816d66c1c28/crates/rattler_networking/src/authentication_storage/storage.rs#L109C1-L109C1)

I'll make an issue there to get this fixed.

**Comment by ruben-arts** [comment]
Yes, it now only happens in debug mode.

**Comment by dhirschfeld** [comment]
Still seeing a lot of noise in `0.4.0` so I assume the fix hasn't made it to a release yet.

![image](https://github.com/prefix-dev/rattler-build/assets/881019/5ba84962-acc8-4c8a-affb-14dc0377e2c3)

**Comment by dhirschfeld** [comment]
Looks like it was fixed in https://github.com/mamba-org/rattler/pull/365

![image](https://github.com/prefix-dev/rattler-build/assets/881019/f4e7e0ae-6223-43c4-b9f9-1d38b0fb9d47)


Which seems to not be included in `0.4.0`:
https://github.com/prefix-dev/rattler-build/blob/3b19118bf4e04733be8e9aaff32addf3546a3414/Cargo.toml#L18
